presto-docs/src/main/sphinx/functions/datetime.rst

```diff
@@ -168,10 +168,10 @@ Specifier Description
 ========= ===========
 ``%a``    Abbreviated weekday name (``Sun`` .. ``Sat``)
 ``%b``    Abbreviated month name (``Jan`` .. ``Dec``)
-``%c``    Month, numeric (``0`` .. ``12``)
+``%c``    Month, numeric (``1`` .. ``12`` [#u]_)
```

Comment:
Maybe use `#z` ("zero") for the note identifier? Since it applies to multiple specifiers (and does not apply to the `u` specifier).